src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -99,6 +99,8 @@ source %{
           return false;
         }
         break;
+      case Op_SelectFromTwoVector:
+        return true;
```

Comment:
Seems not necessary to add one more case here as the default will return true for this case.

Reply:
Logically, yes. It's not necessary, my thought is to make the code explicit, so friendly to read code. But I can remove it if you think it's better to do so.

Reply:
Yes, please. The practice here is to only list the opcodes whose availability would also depend on some other facts like vector size, etc.

Reply:
Sure, fixed.